test/hotspot/gtest/memory/test_arena.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Did you want an SAP copyright?

Reply:
I could add one. Probably should, thanks for noticing.